### PR TITLE
fix: CSAT meter color order

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR includes,
1. Fix for CSAT meter color order issue
2. Fix spacing with thumbnail and agent name

Fixes https://linear.app/chatwoot/issue/CW-2429/fix-csat-meter-color-order

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

**Screenshot**
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/shared/constants/messages.js [hunk 1]
```diff
@@ -58,39 +58,41 @@ export const ALLOWED_FILE_TYPES_FOR_TWILIO_WHATSAPP =
 
 export const CSAT_RATINGS = [
   {
-    key: 'disappointed',
```
Comment (pranavrajs) [inline app/javascript/shared/constants/messages.js:61]: @iamsivin I didn't understand this change. The score 1 is for disappointed faced.

**Comment by sojan-official** [comment]
@iamsivin  could you take a look at the build breakage ?

**Comment by iamsivin** [comment]
> @iamsivin could you take a look at the build breakage ?

@sojan-official Done https://github.com/chatwoot/chatwoot/pull/7805/commits/60666513bf70f17bd511882bd363d37efc26a85d

**Comment by pranavrajs** [comment]
Screenshot 2023-08-27 at 7 38 05 AM

@iamsivin Added a fix for the campaign table. The same component was used there as well.

**Pull request merged**